snippets/javascript.json

```diff
@@ -41,37 +33,32 @@
     "prefix": "hpext",
     "description": "Hapi.js Server Extension",
     "body": [
-      "server.ext('${event}', (request, reply) => {",
-      "\t${1}",
-      "\treturn reply.continue();",
-      "});"
-    ]
-  },
-  "server handler": {
-    "prefix": "hphandler",
-    "description": "Hapi.js Server Handler",
-    "body": [
-      "server.handler('${name}', (route, options) => {",
-      "\t${1}",
+      "server.ext({",
+      "\ttype: '${event}',",
+      "\tmethod: (request, reply) => {",
+      "\t\treturn reply.continue;",
+      "\t}",
       "});"
     ]
   },
   "server inject": {
     "prefix": "hpinject",
     "description": "Hapi.js Server Inject",
     "body": [
-      "server.inject(${options}, (res) => {",
-      "\t${1}",
-      "});"
+      "const res = await server.inject(${options});",
+      "console.log(res.result);"
     ]
   },
   "server initialize": {
     "prefix": "hpinit",
     "description": "Hapi.js Server Initialize",
     "body": [
-      "server.initialize((err) => {",
-      "\t${1}",
-      "});"
+      "const init = async() => {",
+      "\tawait server.start();",
+      "\tconsole.log(`Server running at: ${server.info.uri}`);",
```

Comment:
I think this snippet is for `server.initialize()` not for `server.start()` https://hapijs.com/api#-await-serverinitialize